sniffles/snifflesconfig.py

```diff
@@ -648,6 +665,8 @@ def usage(self):
         print("   sent.  Off by default.")
         print("-b Bi-directional data: Send data in both directions.")
         print("   Off by default.  Automatically sets TCP acks.")
+        # !! Usage for background Traffic
+        print("-B Background Traffic: Percentage of Background traffic to create")
```

Comment:
End with a period.  Also, provide some clue as to what the background traffic will contain (i.e. generic http and ftp traffic).